FBSimulatorControl/Interactions/FBSimulatorInteraction+Bridge.h

```diff
@@ -43,4 +43,7 @@
  */
 - (instancetype)tap:(double)x y:(double)y;
 
+
+- (instancetype)setLocation:(double)x y:(double)y;
```

Comment:
`x` and `y` are poorly named in this context. Use `longitude` and `latitude` throughout.